client/stylesheets/puzzle.scss

```diff
@@ -367,8 +367,53 @@ table.puzzle-list {
     margin-left: 4px;
     position: relative;
 
+    .icon {
+      font-size: 12px;
+      width: 16px;
+      text-align: center;
+      color: $danger;
+      height: 50%;
+    }
+
+    $mutedTop: -4px;
+    $mutedRight: -2px;
+    $deafenDiff: 2px;
+
+    &.muted .muted-icon {
+      position: absolute;
+      top: $mutedTop;
+      right: $mutedRight;
+    }
+
+    &.deafened {
+      color: $secondary;
+      background: #ffffff88;
+      border: $deafenDiff solid white;
+
+      .muted-icon {
+        position: absolute;
+        top: $mutedTop - $deafenDiff;
+        right: $mutedRight - $deafenDiff;
+      }
+
+      .deafened-icon {
+        position: absolute;
+        bottom: 1px;
+        right: $mutedRight - $deafenDiff;
+      }
+    }
+
     .initial {
-      z-index: 10
+      z-index: 10;
+    }
+
+    &.active .initial {
+      // white border around initial
+      // aids readability if the spectrum graph is high
+      text-shadow: -1px 1px 0 white,
```

Comment:
I meant the border to only appear in the active case because the initial was a little hard to read over the blue of the spectrum. IMO the initial has plenty of contrast in the deafened case, but if you disagree I can add it across the board.

Reply:
Sorry, I think I forgot the intent of this someplace between my first and second sentence... I’ll try again:

If spectrum hiding is just done based on the muted flag (see below), then this can just be `:not(.muted)` and we can get rid of the active class entirely.